.prettierrc.json

```diff
@@ -1,12 +1,12 @@
 {
   "arrowParens": "always",
-  "bracketSameLine": true,
+  "bracketSameLine": false,
```

Comment:
<details>

_for my own reference_

You proposed a change to [`bracketSameLine`](https://prettier.io/docs/en/options#bracket-line) with the intent of matching the [`react/jsx-closing-bracket-location`](https://github.com/jsx-eslint/eslint-plugin-react/blob/f83b38869c7fc2c6a84ef8c2639ac190b8fef74f/docs/rules/jsx-closing-bracket-location.md) rule defined in [`standard/eslint-config-standard-jsx/eslintrc.json#L26`](https://github.com/standard/eslint-config-standard-jsx/blob/b21a96c35092f7e70195e5a58c410fc981d2830b/eslintrc.json#L26C1-L26C68)

- the current setting behaves more like `"after-props"`
- the new setting behaves more like `"tag-aligned"`

</details>